scipy/linalg/lapack.py

```diff
@@ -793,6 +793,11 @@
    cgttrs
    zgttrs
 
+   sgtcon
+   dgtcon
+   cgtcon
+   zgtcon
+
```

Comment:
I put this after `?gttrs` because it's after `?gttrs` in the `.src` file. Is there any standard placement for these in `lapack.py` and/or within the `.src` file?

Reply:
Just alphabetical order based on the family name.